c2corg_api/views/feed.py

```diff
@@ -129,11 +129,14 @@ def get(self):
             raise HTTPForbidden('no permission to see the feed')
 
 
-def get_params(request):
+def get_params(request, custom_limit=False):
     lang = request.validated.get('lang')
     token_id = request.validated.get('token_id')
     token_time = request.validated.get('token_time')
-    limit = request.validated.get('limit')
+    if not request.validated.get('limit') and custom_limit:
+        limit = 30
+    else:
+        limit = request.validated.get('limit')
```

Comment:
What about making this:

```python
def get_params(request, default_page_limit=DEFAULT_PAGE_LIMIT):
    lang = request.validated.get('lang')
    token_id = request.validated.get('token_id')
    token_time = request.validated.get('token_time')
    limit = request.validated.get('limit')
    limit = min(
        default_page_limit if limit is None else limit,
MAX_PAGE_LIMIT)
...
```
?

This would make the signature of the function a bit clearer.